types/react-redux/v5/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "redux": "^3.6.0"
```

Comment:
We cannot add `^4.0.0` because the type definitions in `v5` directory are incompatible with Redux v4 which has a breaking change in its `index.d.ts`. That is why I created the new type definitions.

Reply:
Nevermind, I missed the fact there are two `package.json`s